source/server/options_impl.cc

```diff
@@ -72,8 +93,14 @@ OptionsImpl::OptionsImpl(int argc, char** argv, const std::string& hot_restart_v
     exit(1);
   }
 
+  if (max_stat_name_len.getValue() < 127) {
+    std::cerr << "error: the 'max-stat-name-len' value specified (" << max_stat_name_len.getValue()
```

Comment:
Can you add an options_impl death test for this?

Reply:
test added